src/environment/stella_environment.cpp

```diff
@@ -131,7 +131,7 @@ void StellaEnvironment::noopIllegalActions(Action & player_a_action, Action & pl
 
 reward_t StellaEnvironment::act(Action player_a_action, Action player_b_action) {
   
-  double repeat_prob = 0.0;
+  double repeat_prob = 0.25;
```

Comment:
Ok. I'm going to do it as well.

On 24 May 2015 at 11:06, Marc G. Bellemare <notifications@github.com> wrote:

> In src/environment/stella_environment.cpp
> <https://github.com/mgbellemare/Arcade-Learning-Environment/pull/55#discussion_r30953915>
> :
>
> > @@ -131,7 +131,7 @@ void StellaEnvironment::noopIllegalActions(Action & player_a_action, Action & pl
> >
> >  reward_t StellaEnvironment::act(Action player_a_action, Action player_b_action) {
> >
> > -  double repeat_prob = 0.0;
> > +  double repeat_prob = 0.25;
>
> Yes, let's change it to a command line flag before we merge.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mgbellemare/Arcade-Learning-Environment/pull/55/files#r30953915>
> .
>



-- 
Marlos Cholodovskis Machado
http://www.dcc.ufmg.br/~marlos <http://www.cs.ualberta.ca/~machado>